### PR TITLE
Update news.md

### PR DESCRIPTION
**Pull request by neoatomic**

### CODE REVIEW

**Comment by fernandog** [comment]
shotdown

**Comment by neoatomic** [comment]
@fernandog Cant follow.? Did i not understand/explain this correctly in the PR.? 
Little clarification would be appreciated...

File: news.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ All issues and bug reports must be opened at [GitHub](https://github.com/SiCKRAG
 
 <br/>
 
+####2015-10-02####
+
+- Recently we improved the searching of the KAT & TPB Torrent providers. Some users report that they are not able to find new downloads anymore. If that is the case, shotdown SickRage and remove the cache.db file. That should solve the problem.
```
Comment (miigotu) [inline news.md]: `case, shotdown SickRage`

**Comment by fernandog** [comment]
Sorry. Using phone. Keyboard sucks

**Comment by neoatomic** [comment]
Thought you where referring to the PR itself not the Typo. :+1: 
Should be fixed now.

**Pull request merged**